inject.js

```diff
@@ -323,6 +323,12 @@ chrome.extension.sendMessage({}, function(response) {
       }
     });
   }
+ function getArrayFromTag(tagname) {
```

Comment:
We can just used querySelector, no need for the extra gymnastics.

That said, handling audio requires a much deeper rethink of how the plugin works and how the speed is displayed and where. I don't think we should be landing this half-baked.. as it opens up a deep well of technical debt.